### ISSUE TITLE
get_stream_chat not working because of a dependency that doesn't follow Gradle's rules

### ISSUE THREAD

**Issue by level2fast** [GetStream/stream-chat-flutter#2056]
### Which packages are you using?

stream_chat_flutter

### On what platforms did you experience the issue?

Android

### What version are you using?

8.2.0

### What happened?

-  Namespace is missing from Gradle build file for the video_thumbnail dependency: https://pub.dev/packages/video_thumbnail
-  It's being tracked here: https://github.com/justsoft/video_thumbnail/issues/173

### Steps to reproduce

```bash
1. Create a new flutter sample app using vscode
2. Add stream_chat_flutter: ^8.2.0 as an dependency
3. build the apk
...
```

### Relevant log output

```shell
PS C:\Gitlab\getstream-chat-sample-app\get_stream_chat_sample_app> flutter build apk   


FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':video_thumbnail'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s
Running Gradle task 'assembleRelease'...                         1,799ms
Gradle task assembleRelease failed with exit code 1
```

### Flutter doctor output

```shell
PS C:\Gitlab\getstream-chat-sample-app\get_stream_chat_sample_app> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.24.5, on Microsoft Windows [Version 10.0.22631.4541], locale en-US)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.10.1)
[√] Android Studio (version 2024.1)
[√] VS Code (version 1.95.3)
[√] Connected device (4 available)
[√] Network resources

• No issues found!
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by akinsete** [comment]
Am having similar error with this video_player_android plugin.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':video_player_android:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

**Comment by level2fast** [comment]
> Am having similar error with this video_player_android plugin.
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
>   Execution failed for task ':video_player_android:compileDebugJavaWithJavac'.
> 
> > Compilation failed; see the compiler error output for details.

This was fixed in 8.3.0
https://pub.dev/packages/stream_chat_flutter/changelog
